README.md

```diff
@@ -35,6 +35,11 @@ The GrowthBook Proxy repository is a mono-repo containing the following packages
 
 ### What's new
 
+**Version 1.2.1**
+- Move detailed healthcheck statuses to new /healthcheck/status endpoint
```

Comment:
@bryce-fitzsimons Should this be `/healthcheck/checks` as defined [here](https://github.com/growthbook/growthbook-proxy/pull/83/files#diff-1cf6aa85779d76308ecdea0a4d8dd743d8f5a4a7031fa74201c5c2b5563c25e6R72)?

Reply:
oops thanks!